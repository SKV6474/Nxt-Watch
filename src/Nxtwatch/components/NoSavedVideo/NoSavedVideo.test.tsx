import renderer from "react-test-renderer";
import NoSavedVideosRoute from ".";

beforeEach(() => {
  jest.spyOn(console, "warn").mockImplementation(() => {});
});

it("NoSaved Video renders correctly", () => {
  const tree = renderer.create(<NoSavedVideosRoute />).toJSON();
  expect(tree).toMatchSnapshot();
});
