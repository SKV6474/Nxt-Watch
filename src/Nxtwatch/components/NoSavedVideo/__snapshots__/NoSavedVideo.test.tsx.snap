// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSaved Video renders correctly 1`] = `
<div
  className="sc-gHbYXZ fKFcol"
>
  <img
    alt="No Saved videos"
    className="sc-etAVug hvptBg"
    src="https://assets.ccbp.in/frontend/react-js/nxt-watch-no-saved-videos-img.png"
  />
  <div
    className="sc-fGjrnr fMkVhE"
  >
    <div
      className="sc-dsInJy giBYKy"
    >
      No saved videos found
    </div>
    <div
      className="sc-dlUsjx gewwyx"
    >
      you can save your videos while watching them
    </div>
  </div>
</div>
`;
